packages/browser/src/Component/Itowns/3DTiles/3DTilesUtils.js

```diff
@@ -1,5 +1,133 @@
 import * as THREE from 'three';
 
+/**
+ * Create a geometric outliers for each tile of a tileset when its geometry is loaded
```

Comment:
Update description with append (and not create) and wireframe (and not outliers) could be something like:
Traverse a THREE.Object3D and append in children for each Object3D with a geometry a THREE.LineSegment representing its wireframe